dv/management/commands/import_news.py

```diff
@@ -43,11 +43,14 @@ def _save(self, item):
             news.image = item['image'].replace('http://', 'https://')
             news.is_partnership = item['is_partnership'] == 'yes'
             if item['project_id']:
-                news.project_id = item['project_id'].strip()
+                project = Project.objects.filter(code=item['project_id']).exists()
+                if project:
+                    news.project_id = item['project_id'].strip()
```

Comment:
Let's do the following: 
* get the project id (`item['project_id'].strip()`) once, and use it both here and at line 46;
* if project doesn't exist, write a warning message.